BUILD.bazel

```diff
@@ -63,3 +64,24 @@ android_binary(
         "//app",
     ],
 )
+
+# Used for end to end tests
+android_binary(
+    name = "oppia_test",
+    testonly = True,
+    custom_package = "org.oppia.android",
+    enable_data_binding = True,
+    manifest = "//app:src/main/AndroidManifest.xml",
+    manifest_values = {
+        "applicationId": "org.oppia.android",
+        "minSdkVersion": "19",
+        "targetSdkVersion": "29",
+        "versionCode": "0",
+        "versionName": "0.1-alpha",
```

Comment:
Maybe instead: ``0.1-test`` as the version.

Reply:
Done, 
Shouldn't we use `0.1-alpha-test` instead indicating a test version for alpha

Reply:
'alpha' is actually in itself a placeholder that will eventually be removed/changed. 'test' is a longer-term suffix since we always want to indicate this binary is test-only.

We'll eventually have release & beta flavors of the app, not just alpha.